examples/vue/package.json

```diff
@@ -4,11 +4,11 @@
   "private": true,
   "devDependencies": {
     "react-scripts": "0.9.5",
-    "@storybook/vue": "^3.0.0",
+    "@storybook/vue": "3.0.0-alpha.0",
```

Comment:
@ndelangen @alexandrebodin If these deps are filled in by bootstrap, can we use `*`?